src/types/InstantEvent.ts

```diff
@@ -0,0 +1,18 @@
+import { GeneralEvent } from './GeneralEvent';
+//Instant Event type
+export class InstantEvent implements GeneralEvent<string, string> {
+  readonly ph: string = 'i';
```

Comment:
I'm not 100% sure about this, but I think this should be a capital `I` as per https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview#

Reply:
Yeah I found an example on the doc of a Instant Event that uses lowercase 'i', but I will look into which one is correct!

{"name": "OutOfMemory", "ph": "i", "ts": 1234523.3, "pid": 2343, "tid": 2347, "s": "g"}
--
Instant Event Example